dpsim-models/include/dpsim-models/SP/SP_Ph1_NetworkInjection.h

```diff
@@ -39,7 +39,7 @@ class NetworkInjection : public CompositePowerComp<Complex>,
 
   // #### Powerflow section ####
   /// Base voltage [V]
-  Real mBaseVoltage;
+  const Attribute<Real>::Ptr mBaseVoltage;
```

Comment:
I see this is done the same way for the other components, but does `mBaseVoltage` really need to be of `Attribute<Real>::Ptr` type? ls there a specific functionality of Attributes that you'd like to use for `mBaseVoltage`, like logging, interfacing or task scheduling?
If you just want to get the value of `mBaseVoltage` in PFSolver.cpp a more readable solution could be to have a getter function:
```
public:
    // Getter for mBaseVoltage
    Real getBaseVoltage() const {
        return mBaseVoltage;
    }
```